README.md

```diff
@@ -56,12 +56,3 @@ Support us with a monthly donation and help us continue our activities. [[Become
 ## License
 
 Invo is open-sourced software licensed under the [New BSD License][8]. © Phalcon Framework Team and contributors
-
-[1]: https://phalcon.io/
```

Comment:
There are links like `See [CONTRIBUTING.md][7]`

Reply:
Do we need this links here?

Reply:
all these or just this? You just deleted all links from the document

Reply:
Reverted